native/src/modules/app/StoreActionType.js

```diff
@@ -341,7 +341,20 @@ export type SetResourceCacheUrlActionType = {|
   +params: {| +url: string |}
 |}
 
+// Pushes a new snackbar to the queue
+export type PushSnackbarActionType = {|
+  type: 'PUSH_SNACKBAR',
```

Comment:
Should we maybe use `SET_SNACKBAR` here? There can only be one snackbar visible, according to the material design guidelines I suppose. Also `PUSH` is used with our action terminology already in a very specific sense.

Reply:
If we want to keep the array of snackbars, then I would suggest `APPEND` which is a synonym

Reply:
I'd like to keep the array, but `APPEND` would work for me. Would we stick with `POP` or e.g. use `REMOVE` @maxammann ?

Reply:
Renamed to enqueue/dequeue as push/pop/FILO is really used in terms of stacks while this is a queue/FIFO.